src/Graphics-Display Objects/DisplayScreen.class.st

```diff
@@ -271,19 +272,6 @@ DisplayScreen >> replacedBy: aForm do: aBlock [
 	ScreenSave := nil
 ]
 
-{ #category : 'other' }
-DisplayScreen >> restore [
```

Comment:
Wouahhhhh there were reference from DisplayScreen to MorphicUIManager. O_o

Reply:
It's the global callback design pattern :)

Reply:
Yes. And you can build centralized monolithic application pattern :)